idea_town/frontend/static-src/styles/vendor/mozilla-tabzilla/css/tabzilla.scss

```diff
@@ -0,0 +1,34 @@
+#tabzilla {
```

Comment:
Should this and the normalize dep be checked in here? or would we rather have a gulp task which copies them out of `node_modules`?